### DISCUSSION TITLE
/question about yolo model train

### DISCUSSION THREAD

**Discussion by aa851182472** [ultralytics/ultralytics#22367] [Q&A]
Does the pre-trained model of the YOLO series, such as v5, v8, and v11, support training on Huawei Ascend Chip 910B2? I would greatly appreciate it if you could read and respond to this inquiry!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @aa851182472, thank you for your interest in Ultralytics 🚀! This is an automated response — an Ultralytics engineer will also assist you here shortly.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question, please provide as much information as possible, including:
-Your OS and environment (container vs. bare metal)
-Python, PyTorch, and CUDA/CUDNN versions (if applicable)
-The device SDK/driver versions you’re using
-Exact commands you ran and any logs/errors
-And verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) ✅

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
It's not supported. But there's a PR by a user:

https://github.com/ultralytics/ultralytics/pull/22270

**Comment by glenn-jocher** [comment]
Hi @aa851182472, a related Huawei Ascend update: native Ascend export, inference, and validation support has now shipped in Ultralytics. Models can be exported directly to `.om` format with `format=ascend`, and [Ultralytics Platform](https://platform.ultralytics.com) now provides managed exports for Ascend310P1, Ascend310P3, Ascend310B1, and Ascend310B4.

The complete workflow is documented in the [Huawei Ascend integration guide](https://docs.ultralytics.com/integrations/ascend/).

This release does not add training support for Ascend 910B2. Ultralytics can select a single `npu` device through `torch_npu`, but fully validated Ascend training and multi-NPU HCCL DDP remain unsupported.

![Ultralytics Platform Huawei Ascend export](https://gist.githubusercontent.com/glenn-jocher/ddb7e1146039d729ddb891c6bc261e6b/raw/0af8f891222ccd17f783f8185b7c36f690e5b87d/ultralytics-platform-huawei-ascend-export.png)
